randovania/games/dread/json_data/Cataris.txt

```diff
@@ -701,6 +701,8 @@ Extra - asset_id: collision_camera_012
       Any of the following:
           Varia Suit or After Cataris - Move Magnet Walls Button
           Heat/Cold Runs (Intermediate) and Heat Damage ≥ 100
+  > Event - Activate Magnet Walls Button
+      Shoot Diffusion or Wave
```

Comment:
You're gonna need to add a new event node to cover this one. This implies you can get to the above section immediately after pressing the button which will not always be the case especially in door rando

Reply:
I've never done this part but think I got it working correctly, let me know if that isn't the case